cupy/core/_routines_indexing.pyx

```diff
@@ -412,23 +424,44 @@ cdef ndarray _simple_getitem(ndarray a, list slice_list):
     return v
 
 
+_get_index_preamble = '''
+__device__ S get_index(const S* index, bool imcomplete_index, int blk_size,
+                       long long i, long long size) {
```

Comment:
```suggestion
                      int64_t i, ssize_t size) {
```

Reply:
Umm, use of `int64_t` in ElementwiseKernel causes the error below. It seems that `int64_t` is not available as it is. Any workaround for this? 

```
cupy.cuda.compiler.CompileException: /tmp/tmpo3bfsff1/c1046954700a5be1b41b60c419b2a03a_2.cubin.cu(11): error: identifier "int64_t" is undefined
```

Reply:
`long long` is dangerous as it's 32 bits on Windows. You can include the C++ header `<cstdint>` in the preamble, and set a compiler option `-DCUPY_USE_JITIFY` so that Jitify would kick in and resolve `cstdint`.

Reply:
Thank you, @leofang !
Now it is able to build and run in the way you suggested.